packages/pxp/pxp.1.2.9/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+name: "pxp"
+version: "1.2.9"
+maintainer: "codinuum@me.com"
+authors: "Gerd Stolpmann <gerd@gerd-stolpmann.de>"
+homepage: "http://projects.camlcity.org/projects/pxp.html"
+dev-repo: "git+https://gitlab.camlcity.org/gerd/lib-pxp.git"
+bug-reports: "ocaml-pxp-users@orcaware.com"
+license: "MIT/X11"
+patches: "408.patch" {ocaml:version >= "4.08.0"}
+build: [
+  [
+    "./configure"
+    "-without-wlex"
+    "-without-wlex-compat"
+    "-lexlist"
+    "utf8,iso88591"
+  ]
+  [make "all"]
+  [make "opt"]
+]
+install: [make "install" "MANDIR=%{man}%"]
+remove: [
```

Comment:
This is not necessary since opam 2.0, you can remove it along with the `flags: light-uninstall` flag